Sources/BuildServerProtocol/BuildTargets.swift

```diff
@@ -10,6 +10,9 @@
 //
 //===----------------------------------------------------------------------===//
 import LanguageServerProtocol
+import struct Foundation.URL
+
+public typealias URL = Foundation.URL
```

Comment:
We don't need to actually implement the handling for `BuildServerBuildSystem` in this PR, but the BSP protocol layer should use URI as well. I have a patch for this already.